load/load_freebsd.go

```diff
@@ -0,0 +1,279 @@
+// +build freebsd
+
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2016 Intel Corporation
+Copyright 2017 Steven Wills
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package load
+
+/*
```

Comment:
When possible using CGO should be avoided in order to avoid additional dependencies for building, but if it's necessary to make this plugin buildable you need to modify `.sync.yml`.
Following configuration is required:
```
scripts/build.sh:
  cgo_enabled: true
```
https://github.com/intelsdi-x/snap-pluginsync/#scriptsbuildsh